langchain/chains/llm.py

```diff
@@ -29,6 +29,8 @@ class LLMChain(Chain, BaseModel):
     llm: BaseLLM
     """LLM wrapper to use."""
     output_key: str = "text"  #: :meta private:
+    stop: Optional[str] = None
```

Comment:
Sorry for the late response, I didn't realize you had replied because I didn't see a Github notification for this!

The original reason for doing this is that there's not really a way to set different stops for different chains inside a `SequentialChain`. I've pushed a commit that undoes this addition to `LLMChain`, but in return it basically copy-pastes `SequentialChain._call` into `GetMultipleOutputsChain._call`.

LMK what you think. If we do it this way, we can also get rid of `SequentialChain` because we're no longer using it for anything anyways.